conn.go

```diff
@@ -192,6 +195,11 @@ func (c *Conn) flate() bool {
 	return c.copts != nil
 }
 
+// RemoteAddr returns the remote address of websocket connection.
+func (c *Conn) RemoteAddr() net.Addr {
+	return c.remoteAddr
```

Comment:
This should work for both client and server connections, would be good to add tests for both.

Reply:
okkk